faker/providers/ssn/es_MX/__init__.py

```diff
@@ -0,0 +1,250 @@
+# coding=utf-8
+from __future__ import unicode_literals
+
+import random
+import string
+
+from .. import Provider as BaseProvider
+
+
+ALPHABET = string.ascii_uppercase
+ALPHANUMERIC = string.digits + ALPHABET
+VOWELS = "AEIOU"
+CONSONANTS = [
+    letter
+    for letter in ALPHABET
+    if letter not in VOWELS
+]
```

Comment:
I wonder if using sets would make this faster

Reply:
If you think about `list(set(ALPHABET) - set(CONSONANTS))`, this is taking approximately 20% more time, the set conversion comes at a "huge cost" (We speak about some µs). Note, we need to convert it back to a list because we cannot call `random.choice` on sets.